scipy/stats/_continuous_distns.py

```diff
@@ -3505,7 +3505,7 @@ def _cdf(self, x, mu):
         fac = np.sqrt(1.0/x)
         # Numerical accuracy for small `mu` is bad.  See #869.
```

Comment:
```suggestion
        # Numerical accuracy for small `mu` is bad.  See #869.
```
```suggestion
        # Use logs to resolve gh-1396, gh-13614
```
869 referred to Trac 869, which was migrated to gh-1396.